cinema/models.py

```diff
@@ -30,6 +30,10 @@ class Actor(models.Model):
     def __str__(self):
         return self.first_name + " " + self.last_name
 
+    @property
+    def full_name(self):
+        return self.__str__()
```

Comment:
The `full_name` property in the `Actor` model correctly returns the full name by calling the `__str__` method.